### DISCUSSION TITLE
All things marketing

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#178] [General]
Use this thread to present and discuss your marketing ideas and other suggestions that will bring awareness of the ASA Stats project to a wider audience. 

To get a grasp on monetization, please check and join the [ASA Stats Token utilization discussion](https://github.com/asastats/channel/discussions/166) and [ASA Stats API discussion](https://github.com/asastats/channel/discussions/1) for its technical aspects.

Our marketing should target new users (free and professional), as well as third-party developers and companies that will take advantage of our API (for creating new apps and extending the existing apps' functionalities) and/or our user widgets system.

Our main focus right now is creating and financing a Gitcoin bounty for an open-source multiplatform [mobile app](https://github.com/asastats/channel/discussions/166#discussioncomment-3128410) (iOS and Android) that will mirror at least the current ASA Stats website's functionalities.

It is yet to be defined whether we're going to design a central marketing campaign where most of the steps and rules for it would be defined or we're going to lean on a free form. Nevertheless, here comes [a quote from one of our governors](https://discord.com/channels/906917846754418770/917850389930471474/996054567198404658) that brings some basis:

> The first step in "marketing" in any way is determining the message.
> What are we trying to say? 
> What's the value proposition to prospective investors?
> And whom are we speaking to?

Bear in mind that everything in ASA Stats is defined in a public space - every marketing-related DM in any of our channels will be redirected to this thread.

Every idea that adds up will be rewarded. Anybody else too can be rewarded for conducting a presented idea. There's none whatsoever guarantee that your expenses would be reimbursed if you didn't present them in advance.

**DO NOT SPAM PEOPLE ELSEWHERE, SPAM US HERE!!**

**Comment by ipaleka** [comment]
From our [subreddit](https://www.reddit.com/r/asastats/comments/tztje7/comment/igwqsjo/?utm_source=share&utm_medium=web2x&context=3):

- - - - -
May you tell me whether there is another way we can approach and participate in Wageroo on behalf of ASA Stats besides this governance proposal?

- - - - -

We could look at the possibility of a combination special event with Asa stats accepted for wagers. Maybe something algo statistics driven vs a sports event?

**Comment by wxivd** [comment]
### SEO & Content Creation

Aims to increase the visibility, reach, of ASA Stats in organic search (not paid). 

**Main ranking factors**: keywords, content, and links.

### SEO strategy
It is expected from potential candidates to propose a detailed SEO strategy in line with the following:

1- Analytics and Keywords: 
Assess and understand current performance, List of main keywords that prompt ASA Stats appearance in search engine.
* Ascertain current rankings
* Understand website's analytics
* Identify keyword opportunities that is which keywords are in the highest demand within our domain of business. 

2- Website Accessibility and Indexing: 
Ensure that website is being crawled and indexed properly:
* Review content
* link structures
* HTML features (image alt tags).

3- Improve Ranking Factors: 
* keyword density
* HTML markup
* Content: Thus incorporating these keywords into high-quality content.
* External links…etc

4- Competitive Analysis: How ASA Stats rank in comparison to competitors

5- Implementation timeline

**Comment by wxivd** [comment]
### Public Relations

Aims to promote and maintains a public relation strategy

The following aims to provide a structure for PR strategy

**Target Audience**
ASA Stats serves different groups, individual and corporate Investors, Developers and software companies. It is logical to presume people are able to work on parallel, each targeting a specific group within his/her set of expertise.

**Target Media**
Concerning media outlets such as websites, journals, news…etc

**Marketing Concept - Key message**
Since joining our marketing campaign attracts people with different skill sets, the needs of 'the proposal' and 'the target' can accommodate different and equally successful proposals. 

**Examples of PR strategies include:**
- Media outreach
- Press Release
- Editorials
- Events, including virtual events
- Speaking opportunities
- Social Media

**Reply by kerrilija** [reply]
Vestige and/or TinyChart are probably one of the most visited websites in the Algorand ecosystem. They provide an advertisement option - banner on their site. If the price is not too high, I believe it can't hurt. 

Also, a lot of Algorand projects are exposed in [Medium](https://medium.com/) articles.

**Reply by ipaleka** [reply]
> Vestige and/or TinyChart are probably one of the most visited websites in the Algorand ecosystem. They provide an advertisement option - banner on their site. If the price is not too high, I believe it can't hurt.

Yep, if we're going to advertise that way then Vestige's banner is pretty much a central place for a project like ours. I guess we can add other websites if someone mentions them, and that way create a group for ads. 

Another group IMO should be targeted social media ads. Reddit is out of the question (we tried, but starting budget tiers for crypto are out of our reach), so probably Twitter...

> Also, a lot of Algorand projects are exposed in [Medium](https://medium.com/) articles.

Back in 2022., in our DIscord during some other discussion I suggested to @Babbexx22 that he can be a writer for ASA Stats. 

Right now we simply don't have resources for writing blogs, tho afaic everybody who reads the following:

https://github.com/asastats/channel/discussions/178
https://github.com/asastats/channel/discussions/166 
https://github.com/asastats/channel/discussions/130
https://github.com/asastats/docs/blob/main/roadmap.md

will be ready for providing content for a lot of blogs.

**Comment by wxivd** [comment]
### Branding & Advertising

Aims to shape and promote ASA Stats identity and both tangible and intangible qualities. Whereas many may know the services we provide, creating a kind of connection to the brand is key for growth and building momentum

> a brand is a set of expectations, memories, stories and relationships that last, taken together, account for a consumer's decision to chose your product over another - Seth Godin

Two types of Branding
* ASA Stats as future DAO
* ASA Stats as a Service we provide

Key Elements
* Knowledge of current and target audience
* Purpose, Vision
* Value proposition

A successful branding strategy would ential the following
* Research and planning: What is ASA Stats, What does it offer, How this is presented and viewed from target audience perspective, what needs to be done to enhance our position.
* Design: Content types, language used. 
* Creation: build brand through the different sources of multimedia: ads, inforgrafics, data types we share. Where, how, when? 
* Content Marketing
* Testing method to assess outcome. Analytics?

**Comment by wxivd** [comment]
The following aims to gather marketing ideas proposed in Discord

Mcblessing#5551
https://discord.com/channels/906917846754418770/908054304332603402/917783154234892349
 > Hello, so there is a feature I'd like to be implemented,would love to see if the stats of persons ie the number of daily users be shown maybe on the home page before u input your address...


squallara#1991
https://discord.com/channels/906917846754418770/915342738201931836/917759834550583326
> I think it would be a nice idea to start reducing our carbon footprint by becoming carbon negative if possible. I think by doing that, we will get the attention we deserve from Algorand community and foundation since they are already very proud in marketing that Algorand blockchain is already carbon negative. Here are some technology options for start becoming carbon negative: https://www.iea.org/commentaries/going-carbon-negative-what-are-the-technology-options That way we are also doing good for the environment. Maybe we should start planting some trees guys...? What do you think? (edited)


ipaleka#4779
https://discord.com/channels/906917846754418770/917850389930471474/921410345409593374
> Our marketing would be completely focused on the people/companies who would wish they are the only users of the website and that we are focused only on them, nobody else. I mean, such a client will be our persona for every marketing campaign related topics, while our users and the best UX will remain to be the primary everlasting goal of the project


Kenny_Bunkport#7059
https://discord.com/channels/906917846754418770/917850389930471474/921419789300289538
> Sure , all the great sites are first built around free UX but than morph into a platform suited for professional development. I think a DAU and MAU metrics will be essential in the marketing to attract the professional clients.

DAU/MAU Ratio (Daily Active Users to Monthly Active Users ratio) measures how active monthly users are on a daily basis. 

Papa Noff#9216
https://discord.com/channels/906917846754418770/917850389930471474/923327639278714891

> We should reach out to tinycharts to see how much an ad costs or if we could get a discount since we link to them for free 
￼

Mayor Bancelot#5455
https://discord.com/channels/906917846754418770/917850389930471474/996054567198404658
> i think the first step in "marketing" in any way is determining the message.
What are we trying to say? What's the value proposition to prospective investors?
And whom are we speaking to? (audience)


### Members who have expressed interest in marketing campaign
**Graphics designer**
ZAMANI#8965 https://discord.com/channels/906917846754418770/917850389930471474/950854669519970327
James Raven#9579 https://discord.com/channels/906917846754418770/917850389930471474/953452355188322304

**Content creator: video** 
Mayor Bancelot#5455 https://[discord.com/channels/906917846754418770/917850389930471474/996054165971279892](http://discord.com/channels/906917846754418770/917850389930471474/996054165971279892)

**Comment by ipaleka** [comment]
by leaf in [Discord](https://discord.com/channels/906917846754418770/917850389930471474/999854173749252227):
- - - - -
UI update, i think the product can market itself. just needs to an update into the modern age.
like metaverse vibes not microsoft office 

- - - - -
by ipaleka:

Our utilization implies different themes for the website. It is planned that one of those themes replaces the current default/free UI.

**Comment by ipaleka** [comment]
by leaf in [Discord](https://discord.com/channels/906917846754418770/917850389930471474/999854921673342996):
- - - - -
maybe a logo or a theme change during certain holidays or events

**Reply by ipaleka** [reply]
This is a good suggestion, tho imo we should probably wait for this until the internalization is implemented.

**Comment by kerrilija** [comment]
**ASA Stats Utility NFT**

As we know, words about NFTs spread like crazy. If it has a utility, even better. We can initiate a creation of a utility NFT. 
I was picturing something like this. They can be bought with ASASTATS token(after whitelist spots etc.)

Wallets holding a...

**Common**(30 items) for a price of 50 Algo equivalent:

- can change the appearance of the app(skins, etc.)


**Uncommon**(10 items) for a price of 150 Algo equivalent:

- common features + get real-time total value updates(no refresh needed)


**Legendary**(2 items) for a price of 1500 Algo equivalent:

- get a life time subscription and a governor seat


If you like or support the idea, please add your suggestions, or give an opinion if you don't. We can hire a NFT creator or there maybe is one in our community?  We can also make a twitter giveaway for an NFT.

**Reply by marucat27** [reply]
I do believe the creation of NFTs will especially with utility will gain attention of the community.  Mint size will be important also.  Low mint count will for sure make it desirable but with less total funds received.  What subscription tier were you thinking of the legendary NFT?

**Reply by kerrilija** [reply]
Everything is to be discussed. We need to find a balance between being generous and not doing damage to our services. We have to calculate what is the cost of having a subscripted user (how much resources is he or she leveraging), so we can be more accurate in terms what premium features can we offer for free. I've put random values/numbers. Maybe you or someone with more experience in NFT in general can provide better item numbers, rarity options etc.

**Reply by ipaleka** [reply]
This goes to the same category as [this Twitter discussion](https://twitter.com/asastatscom/status/1573295060271370240) reveals:

- - - - -
By mochanerd.algo
·
I just wish being an LP would count towards a subscription Tier! Like Defly for example
- - - - -
Our utilization document and lifetime policy brought in forbids any other currency/token to be used for subscription payments besides ASASTATS.
Nevertheless, we'll for sure sponsor farming programs for the providers who want to spend rewards to purchase our subscription tiers. :)
- - - - -
I meant ASASTATS-ALGO LP
- - - - -
Yes, of course, it's just that our Permission ASC will accept only ASASTATS (ID 393537671) and the DAO will be legally obliged to secure that. Actually, that will be the only condition for DAO to take control over the project.

https://github.com/asastats/channel/wiki/ASA-Stats-Token-utilization#the-first-dao-voting-session
- - - - -

In short and as of today, a subscription tier purchased by ASASTATS is the only way. Every suggestion for change that policy should be proposed in the [utilization discussion](https://github.com/asastats/channel/discussions/166). Once when we change our utilization document we can apply such a changed policy to other parts of the project like in this thread/discussion.

Also, we've got a dedicated discussion for the ASA Stats NFTs: https://github.com/asastats/channel/discussions/111, so I'm going to copy this suggestion there.

Last but not least, this doesn't mean that your idea is bad, at contrary, but we have to bypass [The](https://github.com/asastats/channel/wiki/ASA-Stats-Token-utilization#the-first-dao-voting-session) condition somehow, for example by staking such an NFT in order to get ASASTATS used to purchase a subscription and that way accomplish what you have proposed.

**Comment by ipaleka** [comment]
https://twitter.com/asastatscom/status/1581277617184915456
![tweet](https://user-images.githubusercontent.com/49662536/197510822-8b674a0e-6f39-4b04-a88d-66bca3f70be5.png)
